INeedCoffee/CoffeeShop/CoffeeShop.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace INeedCoffee
+{
+    public class CoffeeShop
+    {
+        public string Name { get; set; }
+        public double Latitude { get; set; }
+        public double Longitude { get; set; }
+        public double DistanceToUser { get; set; }
```

Comment:
Yes, it definitely can exist without a distance and it makes sense it will cause issues if we want to scale up our application. I will make sure to document myself properly to avoid situations like this.